src/backlight/positions/positions.py

```diff
@@ -83,7 +86,11 @@ def calc_positions(
     pos = Positions(_pricer(trades, mkt, principal))
     pos.reset_cols()
     pos.symbol = trades.symbol
-    pos.currency_unit = trades.currency_unit
+
+    pos.currency_unit = None
+    if trades.currency_unit:
+        pos.currency_unit = trades.currency_unit
```

Comment:
How about just replacing 3 lines as `pos.currency_unit = trades.currency_unit`?
It does not work?

Reply:
I used this because at this moment in most tests and notebooks the `currency_unit` wasn't defined for trades. But you are right, I will replace it and fix bugs if there is.